src/back/glsl/mod.rs

```diff
@@ -3580,6 +3605,7 @@ const fn glsl_built_in(built_in: crate::BuiltIn, output: bool) -> &'static str {
                 "gl_FragCoord"
             }
         }
+        Bi::ViewIndex if multiview_ovr => "int(gl_ViewID_OVR)",
```

Comment:
What's the purpose of having two branches if we always take the first one?

Reply:
Oops, forgot to have this take `is_webgl` as input.,